benchmarking/generate_users.py

```diff
@@ -0,0 +1,67 @@
+"""
+generates dummy users by sending multiple concurrent requests to signup API
+stores the user details like UserID, Username, EmailID, MobileNo, Password,
+Response Time and Response Status Code to a CSV file
+"""
+
+import concurrent.futures
+import random_string
+import requests
+import csv
+
+USERNAME_MIN_LENGTH = 4
+USERNAME_MAX_LENGTH = 8
+PASSWORD_LENGTH = 8
+MOBILE_NUMBER_LENGTH = 10
+EMAIL_DOMAIN = "@dummy.com"
+ITERATIONS = 1000000
+OUTPUT_FILE_NAME = "users.csv"
+CSV_COLUMN_NAMES = ["UserID", "Username", "EmailID", "Password", "MobileNo", "Response Time"]
+
+
+def signup_request():
+    username = random_string.alpha_numeric_variable_length(USERNAME_MIN_LENGTH, USERNAME_MAX_LENGTH)
+    email = username + EMAIL_DOMAIN
+    mobile_no = random_string.numeric_fixed_length(MOBILE_NUMBER_LENGTH)
+    password = random_string.alpha_numeric_fixed_length(PASSWORD_LENGTH)
```

Comment:
Since this is test data only, let's make password some function of username so that we can easily calculate it instead of it being random. WDYT?

Reply:
We are storing the passwords in `users.csv` so we can retrieve them easily. Should we change and instead of storing password, compute it based on the username?

Reply:
sure. That's fine.